browser/resources/brave_new_tab_page_refresh/components/news/news_feed.tsx

```diff
@@ -22,6 +22,12 @@ export function NewsFeed() {
     setTimeout(() => setShouldRenderNews(true), 1000)
   }, [])
 
+  React.useEffect(() => {
+    const handleScroll = () => braveNews.setShouldRenderImages(true)
+    document.addEventListener('scroll', handleScroll, { once: true })
+    return () => document.removeEventListener('scroll', handleScroll)
+  }, [braveNews.shouldRenderImages])
```

Comment:
Hey, I wonder if this should just live in `Context.tsx`, as it doesn't really depend on anything in this file.

Then we could remove the `setShouldRenderImages` from the context (as nothing outside there would ever set it).

Also, if we don't this will break all the images in the old New Tab Page

Reply:
I suggested moving the scroll detection here, since News is presumably designed to work standalone as well?

Reply:
The fade in already isn't going to work in standalone (and we don't have a standalone :laughing:), so I think better to put it in the Context for now. 

If we want it to work in Standalone, we'll probably need to revisit how the fade works anyway